analysis/deep2_r23_o32/curvefit_analysis.py

```diff
@@ -68,6 +81,19 @@ def run_experiment_Zcal(fitspath, fitspath_curvefit, fitspath_ini, n_bins=4,
     lR23 = gx^3 + ax^2+ bx+ c
     """
     suffix = ''
+    if threevariable:
+        suffix += 'threeparam_'
+    else:
+        suffix += 'twoparam_'
+
+    if secondorder:
+        suffix += 'secondorder'
+    else:
+        suffix += 'thirdorder'
+
+    if include_rlimit:
+        suffix += '_includerobust'
+
```

Comment:
This is significant improvements as you probably noticed. It's easier to read. My only suggestion is to define `prefix` and then merge the two. That gives us flexibility should we only need `prefix`.

Reply:
fixed